third_party/libjpeg-turbo/turbojpeg.c

```diff
@@ -44,17 +44,13 @@
 #include "./cdjpeg.h"
 
 #ifdef STARBOARD
-#include "starboard/client_porting/poem/strings_poem.h"
-#include "starboard/client_porting/poem/string_poem.h"
+#include "starboard/common/log.h"
 #include "starboard/configuration.h"
-#include "starboard/client_porting/poem/stdio_poem.h"
 
```

Comment:
Is this empty line useful somehow?
If not, maybe remove it only if you're making some other amendment anyway.